src/components/Announce/blogpost.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
https://github.com/odota/blog/pull/7 we can remove the release label or something to hide it later.   Maybe you can test without the closed query param since I don't actually want to merge it :)